projects/igniteui-angular/src/lib/core/styles/components/grid/_excel-filtering-component.scss

```diff
@@ -10,6 +10,10 @@
             @extend %igx-excel-filter__loading !optional;
         }
 
+        @include e(sizing) {
+            @extend %igx-excel-filter__sizing !optional;
+        }
```

Comment:
Why do we need a new class selector? Can't we simply use the existing %grid-excel-filter?

Reply:
If you take a look at the usage of the newly introduced class:
```
    @HostBinding('class.igx-excel-filter__sizing')
    protected get shouldApplySizes(): boolean {
        return !(this._minHeight || this._maxHeight);
    }
```
we set this only if we do not have neither minHeight nor maxHeight set from the inputs of the ESF. If we use the exising main css class %grid-excel-filter we cannot do this unless we set min/maxHeight as inline styles that override the default behaviour of the sizing mixin which I don't really like